ADAL/src/validation/ADAuthorityValidation.m

```diff
@@ -23,36 +23,48 @@
 
 
 #import "ADAuthorityValidation.h"
-#import "ADAuthorityValidationRequest.h"
+
+#import "ADAadAuthorityCache.h"
 #import "ADDrsDiscoveryRequest.h"
+#import "ADAuthorityValidationRequest.h"
 #import "ADHelpers.h"
 #import "ADOAuth2Constants.h"
 #import "ADUserIdentifier.h"
 #import "ADWebFingerRequest.h"
+
 #import "NSURL+ADExtensions.h"
 
+#include <pthread.h>
```

Comment:
It isn't. I originally had the cache as part of this file and then refactored it out.